design/FY2017/NFP-UpdateLEEDSummaryReport.md

```diff
@@ -0,0 +1,1386 @@
+ - With details as required by the U.S. Green Building Council (USGBC), including the information needed for the Energy Performance Calculator and Table 1.4 spreadsheets. 
+
+In addition, the current outputs from the LEED Summary Report shall be revised to be consistent with the requirements of the USGBC.
+
+
+## Approach ##
+
+The main approach used was to reach out to stakeholders and to review the Minimum Energy Performance Calculator spreadsheet. This approach resulted in more feature requests than could be performed within the budget. Below are suggestions from stakeholders and an indication if they are included in this NFP.
+
+Summary of a call with USGBC representatives:
+
+- Update the performance results including new end uses (included). 
+- Old EAp2-1 style labels are not longer used. Should just use names of tabs and subtable names (included).
+- Gathering results now scattered through out current tabular results file more valuable to LEED users that deriving new outputs for other tabs of Minimum Energy Performance Calculator calculator (included).
+
+- Importing. Best option would be for utility to automatically populate Minimum Energy Performance Calculator spreadsheet based on EnergyPlus inputs and outputs (EnergyPlus or OpenStudio team would maintain the custom spreadsheet). Second best option would be make importing results as simple as possible (USGBC would maintain spreadsheet and importing formulas). A special CSV would make this more repeatable. (not included)
+- Flag zones that are signficantly different maximum heating and cooling loads for Appendix G3.1.1c (not included). 
+- Economizer use status (not included). 
+- Peak operating vs. design capacity and timestamp of peak (not included). 
+- When is optimum start used (not included).
+
+In addition to the USGBC call, numerous email exchanges with various users and stakeholders suggest the following changes:
+
+ - Demand for each end use should be the peak demand for the individual end use (included).
+ - New custom end uses will be attached to end use subcategory user input so that users can categorize (included).
+ 
+ - Allow for import of EnergyPlus results to Minimum Energy Performance Calculator spreadsheet (not included).
+ - Enhance the AppGPostProcess software to compare the proposed with the four rotated buildings and report the savings and LEED points (not included).
+ - Aggregate the areas based on program type for use in the space by space lighting and equipment power density summaries. Add a space program type field into the zone object to allow for such aggregation of results (not included).
+ - Add an airside equipment summary for each air handling unit instead of results across coils and fans (not included).
+ - Add fenestration assembly u-value and SHGC including edge and frame effects (not included).
+ - Add end use subcategory fields to objects that do not currently have them (not included).
```

Comment:
Isn't this what you are describing below? To add End Use Subcategory fields to chillers, boilers, cooling towers?